documentation/Database-Schema-Permission.md

```diff
@@ -0,0 +1,34 @@
+# Database Schema Permission
+
+![Database Schema](images/database-schema-permission.png)
+
+## Database Area description: Permission
+
+### Definition
+To define permissions with actionables for the user over an application templates.
+
+We like to define the permissions definition using: Who, how and what.
+Who => Is the `permission_name`, which describes the type of action to be associated with users.
+How => Is the `permission_policy` or how the user that has this permission can interact with an application template (through actions).
+What => Is the `template_permission` that connects some permission with an application template or section of an application template.
+
+Before any user is included, permissions can be created. The permission definition consists only on the `permission_policy` and the `permission_name`. And then what links this permission with an application template is the `template_permission`, and after the permission is granted to a user (or a user in a company) what links this permission with an user is the `permission_join`.
+
+If the user is grated a permission to do a certain action on an applications/sections of the application, this action will be displayed in the system. There are also SQL policies that get created to restrict areas of the database that users have access to (based on the same permissions).
+
+## Tables
+* **permission policy**
```

Comment:
I see you've changed the other ones to Subheadings. Good idea, just a reminder to match here too. :)

Reply:
Done